### DISCUSSION TITLE
File name date

### DISCUSSION THREAD

**Discussion by LeoMajor1** [gildas-lormeau/SingleFile#1372] [Q&A]
Hi,

This is simple and i want to keep it short:
  Whenever i save a page, the name is something like 21.01.2024 18_47_24 now and before was like 2024-01-03 17_48_32
  
Q: Where do i change anything to have it like it was before? Can i change anything from here?
                  "filenameTemplate": "%if-empty<{page-title}|No title> ({date-locale} {time-locale}).{filename-extension}",

**Comment by gildas-lormeau** [comment]
Were you using your computer in a different country than before? It seems that you've moved a time zone and that the way dates are formatted in this location is not the same.
